from_mwt_ds/DataScience/vw_executor/vw.py

```diff
@@ -125,10 +140,87 @@ def __init__(self,
+        import os
+        from datetime import datetime
+
+        if os.name == "nt":
+            def symlink_ms(source, link_name):
+                from subprocess import call
+                call(['cmd', '/C','mklink', link_name, source], shell=False, stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
+            os.symlink = symlink_ms
+        
+        def remove_argdash(arg: str):
+            if len(arg) > 1 and arg[0] == "-" and arg[1] == "-":
+                return arg[2:]
+            elif arg[0] == "-":
+                return arg[1:]
+            else:
+                return arg
+        
+        def clean_args(self):
+            args = self.job.name.split()
+
+            args = [remove_argdash(arg) for arg in args]
+            return ".".join(args)
+        
+        argdirname = clean_args(self)
+        if not current_time:
+            current_time = datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+        
+        if self.input_file.parent == self.input_file:
+            raise ValueError("Input files cannot be on the root folder")
+
+        input_file_dir = self.input_file.parent.absolute()
+        input_file_name = str(self._order_position) + "th_file"
+
+        stderr_temp = self.stdout.path.parent / (self.stdout.path.name + '.pending')
+
+        if self.status == ExecutionStatus.Failed:
+            assert stderr_temp.exists()
+            human_task_dir = Path.cwd() / "human" / current_time / "ERRORS" / argdirname / input_file_dir.name / input_file_name
+        else:
+            human_task_dir = Path.cwd() / "human" / current_time / argdirname / input_file_dir.name / input_file_name
+        human_task_dir.mkdir(parents=True, exist_ok=True)
+
+        def create_symlink_if_exists(source:Path, link_name):
+            if source.exists():
+                os.symlink(source, link_name)
+
+        create_symlink_if_exists(self.stdout.path.absolute(), human_task_dir / "stdout.txt")
+        create_symlink_if_exists(self.input_file.absolute(), human_task_dir / self.input_file.name)
+
+        stdout_file = self.stdout.path.parent / (self.stdout.path.name + '.out.txt')
+        create_symlink_if_exists(stderr_temp.absolute(), human_task_dir / "ERROR_stdout.txt")
+
+        if stderr_temp.exists():
+            create_symlink_if_exists(stdout_file.absolute(), human_task_dir / "more_ERROR_stdout.txt")
+        else:
+            create_symlink_if_exists(stdout_file.absolute(), human_task_dir / "more_stdout.txt")
+
+        for output_arg, filename in self.outputs.items():
+            create_symlink_if_exists(filename.absolute(), human_task_dir / translate_output[output_arg])
+
+        if self.model_file:
+            create_symlink_if_exists(self.model_folder.joinpath(self.model_file).absolute(), human_task_dir / "input_regressor.vwmodel")
+        
+        def rename_outputs(self):
+            splitted = self.args.split()
+            for i in range(len(splitted)-1, 0, -1):
```

Comment:
that kind of iteration is a bit hacky: for example it will break if i have input file named "-i": it is weird but legit.
At the same time task have access to structured information about outputs.
So, looks like something like ChainMap(job.opts, {'-i': 'input_regressor'}, {o: translate_output[o] for o in self.outputs}) is more robust here?